client/js/socket-events/msg.ts

```diff
@@ -105,7 +105,11 @@ function notifyMessage(
 		return;
 	}
 
-	if (msg.highlight || (store.state.settings.notifyAllMessages && msg.type === "message")) {
+	if (
+		channel.notifyAll ||
```

Comment:
This triggers on way too many things no?

Do you *really* want a notification on every single join / away / topic / mode and what have you?